internal/cli/atlas/quickstart/dbuser_setup.go

```diff
@@ -97,3 +96,13 @@ func (opts *Opts) newDatabaseUser() *atlas.DatabaseUser {
 		Username:     opts.DBUsername,
 	}
 }
+
+func GeneratePassword() (string, error) {
+	const passwordLength = 12
+	pwd, err := randgen.GenerateRandomBase64String(passwordLength)
+	if err != nil {
+		return "", err
+	}
+
+	return pwd, nil
```

Comment:
[q] does `randgen.GenerateRandomBase64String(passwordLength)` returns and empty string on error? if yes this can be simplified to 
```suggestion
	return randgen.GenerateRandomBase64String(passwordLength)
```

Reply:
no, it returns an error